examples/meta/src/base_api/put_get_add.sg

```diff
@@ -33,6 +33,8 @@ Machine svm = machine("LibSVM")
 svm.put("kernel", k)
 Kernel k2 = as_kernel(svm.get("kernel"))
 svm.put("kernel", k2)
+svm.put("libsvm_solver_type", 2)
```

Comment:
Should put a comment saying that both are the same

Reply:
OK! Btw is it possible to test if get is working? I guess I need to add `get_option` to the parser?

Reply:
Yes should be. Just via changing example and the target Json filed to potentially replace some method call strings